### DISCUSSION TITLE
Create tasks counting Pomodoros

### DISCUSSION THREAD

**Discussion by AncapBitcoin** [focustimerhq/FocusTimer#587] [Ideas]
A good tip would be the possibility to create individual tasks, and for each task the count of how many Pomodoros were performed.

**Comment by borisservo** [comment]
Hey good morning, is this idea still on?

**Reply by kamilprusko** [reply]
I'd like an agenda view, with tasks and events; and history view in place of simple stats at one point.

To count the number of pomodoros, first it needs refactoring how pomodoros are regarded as completed. In wip/gtk4 branch the threshold is set at 80% of initial duration. Long pomodoros will be counted as several pomodoros.

Port to GTK4 needs to be done before adding new elements to the UI. Work on the task list I could start in 2024 at the earliest.
